app/autoload.php

```diff
@@ -1,29 +1,8 @@
 <?php
 
-/**
- * Copyright 2015 SURFnet B.V.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-use Doctrine\Common\Annotations\AnnotationRegistry;
 use Composer\Autoload\ClassLoader;
 
-/**
- * @var ClassLoader $loader
- */
+/** @var ClassLoader $loader */
 $loader = require __DIR__.'/../vendor/autoload.php';
```

Comment:
No need to write the loader to a variable, we can return it directly

Reply:
Indeed, removed the declaration